cv_next/app/feed/components/filterPanel.tsx

```diff
@@ -20,12 +20,12 @@ export const FilterPanel = ({
   onChange: (filters: filterValues) => void;
 }) => {
   const [searchValue, setSearchValue] = useState(defaultFilters.searchValue);
-  const [categoryId, setCategoryId] = useState(defaultFilters.categoryId);
+  const [categoryId, setCategoryId] = useState(defaultFilters.categoryId)
```

Comment:
Bad prettier config, run again on all the PR files with the proper configuration.